ansible/environments/group_vars/all/vars.yml

```diff
@@ -7,6 +7,9 @@ TIME_ZONE: "Etc/UTC"
 # Update Openstack dist code name like ussuri etc.
 OPENSTACK_DIST: wallaby
 
+# Specify the controller hostname where wlm-cron should run
+CONTROLLER_HOSTNAME: ""
```

Comment:
Please rename it as "WLM_CRON_CONTROLLER_HOSTNAME"

Reply:
okay.